KDIC/Article.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  Article.swift
+//  KDIC
+//
+//  Created by Shaun Mataire on 4/24/16.
+//  Copyright © 2016 Colin Tremblay. All rights reserved.
+//
+
+import Foundation
+
+
+class Article{
+    var articleTitle: String
+    var author: String
+    var datePosted: String
+    var articleText: String
+    var articleUrl: String
+    
+    
+    init(_ articlesArray: Dictionary<String, String>){
```

Comment:
Maybe `articleDictionary` or simply `dict` or the like? It's categorically not an array at all.